membership-attribute-service/conf/routes

```diff
@@ -9,26 +9,16 @@ GET        /user-attributes/me/mma-membership               controllers.AccountC
 GET        /user-attributes/me/mma-monthlycontribution      controllers.AccountController.monthlyContributionDetails
 GET        /user-attributes/me/mma-paper                    controllers.AccountController.paperDetails
 
+# IMPORTANT: for update endpoints (e.g. POST) to correctly support OPTIONS for CORS they must be added to AppLoader.scala
```

Comment:
It seems like this comment only makes sense if we leave the manually maintained list in AppLoader.scala